src/main/webapp/assignPanel.js

```diff
@@ -0,0 +1,272 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* eslint-disable no-unused-vars */
+/* global getTaskListsAndTasks, postNewTaskList,
+ enableAssignStartResetButton, createTextListElementFromString,
+ encodeListForUrl, AuthenticationError, Task, postNewTask,
+ disableAssignAcceptRejectButtons */
+
+// Script for handling the behaviour of the Assign panel's features
+
+const assignTaskListTitle = 'Actionable Emails (Generated By Blueprint)';
+let assignTaskListId;
+let assignTasks = [];
+let nDays;
+let unreadOnly;
+let subjectLinePhrases = [];
+let actionableEmails = [];
+
+const taskNotesPrefix =
+    '*** ' +
+    'Made automatically with Blueprint. ' +
+    'Please add notes above this line, ' +
+    'and do not edit anything below this line. ' +
+    'Blueprint needs this information to better serve you. ' +
+    'Thank you for your understanding. ' +
+    '***' +
+    '\n';
```

Comment:
Use template literals here

Reply:
Same reason as above for not using template literals